### PR TITLE
Move docker files in project

### PR DESCRIPTION
**Pull request by bersace**
Since we now have only one Dockerfile, it's legit to maintain it with temboard. Especially with all the hard renaming we are doing.

Before testing:

``` console
$ docker-compose down -v
$ make -C docker clean build
```

### Testing quickstart

Follow `QUICKSTART.md`

Clean with:

``` console
^C
$ docker-compose down -v
```

### Testing docker entrypoint dev env

``` console
$ cd docker/
$ make dev
$ docker-compose exec ui bash
# gosu temboard docker-entrypoint.sh
```

Clean the same way.

### Testing UI dev

Follow `CONTRIBUTING.md`.

### CODE REVIEW

**Comment by pgiraud** [comment]
When I run `make dev` on my machine I get the following error:
"ERROR: for ui  No such image: sha256:ae5e3f09526c4c7f4354cf"

**Comment by bersace** [comment]
@pgiraud what tells `docker-compose ps` ?

Anyway, just trash your compose project with `docker-compose down`.

File: docker/docker-compose.quickstart.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    links:
+      - repository
+    environment:
+      PGHOST: repository
+      PGUSER: temboard
+      PGPASSWORD: temboard
+    ports:
+      - "8888:8888"
+    links:
+      - repository
```
Comment (pgiraud) [inline docker/docker-compose.quickstart.yml]: Duplicated

File: docker/docker-compose.dev.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+version: '3'
+
+services:
+  ui:
+    volumes:
+      - ./docker/entrypoint.sh:/usr/local/bin/docker-entrypoint.sh
```
Comment (pgiraud) [inline docker/docker-compose.dev.yml]: should be:
./entrypoint.sh:/usr/local/bin/docker-entrypoint.sh

Comment (bersace) [inline docker/docker-compose.dev.yml]: That looks ok:

``` json
        "Mounts": [                                   
            {                                         
                "Type": "bind",
                "Source": "/home/bersace/src/dalibo/temboard-agent-dev/docker/entrypoint.sh",
                "Destination": "/usr/local/bin/docker-entrypoint.sh",
                "Mode": "rw",
                "RW": true,                                                                 
                "Propagation": ""
            },                         
  ```

What error do you encounter ?

Comment (bersace) [inline docker/docker-compose.dev.yml]: Fixed. Thanks :heart:

File: docker/docker-compose.quickstart.yml (hunk 1 above)
Comment (bersace) [inline docker/docker-compose.quickstart.yml]: Thanks :+1:

**Pull request merged**